node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/Limits.java

```diff
@@ -17,7 +17,7 @@
 
     private final ClusterResources min, max;
 
-    private Limits(ClusterResources min, ClusterResources max) {
+    public Limits(ClusterResources min, ClusterResources max) {
```

Comment:
Need to verify that either both arguments are `null` or neither and generalize `isEmpty()`